UPGRADE-7.1.md

```diff
@@ -65,6 +65,24 @@ as service.
 
 All service classes that are loaded by `ServiceLocator::get()` must now be `public="true"` in their service definitions.
 
+## Correct Cronjob Hierarchy
+
+All Cron Jobs (instead of TCMSCronJob or TdbCmsCronjobs) should now call the superclass constructor.
```

Comment:
"

Cronjob classes need to extend the class TdbCmsCronjobs. Currently the often used class TCMSCronJob does not have all the necessary fields.

So searching for "TCMSCronJob" in the project code should give you a complete list for this task.

Furthermore every cronjob should only calls the parent constructor like so...